pkg/spec/spec.go

```diff
@@ -328,6 +335,42 @@ func blockAccessToKernelFilesystems(config *CreateConfig, g *generate.Generator)
 	}
 }
 
+// systemd expects to have /run, /run/lock and /tmp on tmpfs
+// It also expects to be able to write to /sys/fs/cgroup/systemd and /var/log/journal
+
+func setupSystemd(config *CreateConfig, g *generate.Generator) error {
+	mounts, err := config.GetVolumeMounts([]spec.Mount{})
+	if err != nil {
+		return err
+	}
+	options := []string{"rw", "private", "noexec", "nosuid", "nodev"}
+	for _, dest := range []string{"/run", "/run/lock", "/sys/fs/cgroup/systemd"} {
+		if libpod.MountExists(mounts, dest) {
+			continue
+		}
+		tmpfsMnt := spec.Mount{
+			Destination: dest,
+			Type:        "tmpfs",
+			Source:      "tmpfs",
+			Options:     append(options, "tmpcopyup", "size=65536k"),
```

Comment:
tmpcopyup is not standard OCI, it is understood only by runc, could these be just empty tmpfs?

Reply:
No and we already do this for tmpfs.  I think the oci runtimes should ignore it.
The problem with not copying up is containers that come with a subdir of /run/httpd will blow up if the tmpfs does not have the httpd directory copied up.